### DISCUSSION TITLE
azure 建立免費帳戶

### DISCUSSION THREAD

**Discussion by pocper1** [1101-GS3073-C/Python-Community-GS3073-C#42] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/76590438/136657007-3b106f2c-ae1b-4b56-a03b-51f18c3878cf.png)
![image](https://user-images.githubusercontent.com/76590438/136656972-c94d5ba9-2fc3-4be2-af2e-33f73f03dd08.png)
在註冊azure 免費帳戶會遇到要輸入信用卡資訊，請問信用卡是一定要輸入的嗎?
如果沒有信用卡要怎麼辦呢?

**Comment by chia-kai-chang** [comment, marked as answer]
你要用學校的office365帳號登入azure，不用輸入信用卡。

**Reply by pocper1** [reply]
處理好了，因為我在導向微軟azure 申請帳號時，要選擇教育版的帳號開通，而不是一般版帳號，謝謝老師的回覆。
